docs/user_guide/1_introduction.md

```diff
@@ -0,0 +1,61 @@
+## Introduction
+
+This manual describes the use and function of the Shadow Modular Grasper
+
+### Abbreviations
+
+Abbreviation | Meaning
+--- | ---
+API | Application Programming Interface
+DoF | Degrees of Freedom
+EC | EtherCAT
+EEPROM | Electrically Erasable Programmable Read Only Memory. 
+GUI | Graphical User Interface.
+LED | Light Emitting Diode. (A small coloured light)
+MCU | Micro Controller Unit. (A small, usually embedded, CPU)
+PC | Personal Computer.
+PWM | Pulse Width Modulation. (The digital method used to emulate an analogue signal.)
+ROS | Robot Operating System.
+
+### Modular Grasper Technical Spec
+
+The Shadow Modular Grasper is an advanced solution for grasping and handling a wide range of objects.
+
+<ul>
+<li>9 Degrees of Freedom (DoF) <img align="right" src="https://raw.githubusercontent.com/shadow-robot/fh_documentation/F_SRC-1674_integrate_latest_changes/docs/img/tech_spec_1.png" alt="Technical Specification 1" style="max-width:100%;"></li>
+<li>3 Fingers</li>
+<li>3 Joints per Finger
+<ul>
+<li>Base rotation,	-45° to +45°</li>
+<li>Proximal,	-80° to +60°</li>
+<li>Distal,		-60° to +80°</li>
+</ul>
+</li>
+<li>Maxon DC Motors</li>
+<li>Backdrivable Gearbox</li>
+<li>Non-slip surfaces for Grasp Stability</li>
+<li>2.7 kg weight</li>
+<li>2 kg Payload (using power grasp)</li>
+<li>20 N intermittent force per fingertip, 10 N continuous</li>
+<li>260 mm maximum span</li>
+<li>1 kHz Position Control Loop , with 0.02° position sensing resolution</li>
+<li>1.39 mNm resolution direct Torque sensing at each joint</li>
```

Comment:
In theory maybe... Should we add "up to" or something to cover our ar...behinds?

Reply:
@RobShadow what do you think?

Reply:
10N has definitely been shown, the "intermittent" is the covering term :)

Reply:
Ok, I withdraw my comment.